lib/msal-core/tests/MSALSpec.js

```diff
@@ -116,20 +116,50 @@ describe('Msal', function () {
         };
         window.localStorage = storageFake;
         window.sessionStorage = storageFake;
-        // Init adal 
+        // Init adal
 
         global.window = window;
         global.localStorage = storageFake;
         global.sessionStorage = storageFake;
         global.document = documentMock;
         global.Math = mathMock;
 
-        msal = new MsalModule.UserAgentApplication("0813e1d1-ad72-46a9-8665-399bba48c201", null, function (errorDes, token, error) {
+        // Setting a default value in JSON format
+        var config = {
+            "auth": {
+                "clientId": "0813e1d1-ad72-46a9-8665-399bba48c201",
```

Comment:
we need more tests to have good coverage of tests using various combinations of input params.

Reply:
Yes. Probably next week will get a separate PR out for this.

Reply:
Resolving for merge.